parser/scanner.l

```diff
@@ -635,6 +637,7 @@ year_month		{y}{e}{a}{r}_{m}{o}{n}{t}{h}
 			return abs
 {add}			return add
 {adddate}		return addDate
+{admin}			return admin
```

Comment:
admin and ddl are not mysql reserved keyword, should set `lval.item = string(l.val)`